openapi.json

```diff
@@ -1802,7 +1851,7 @@
     "/credentials/basic": {
       "get": {
         "summary": "HTTP Basic authentication",
-        "description": "This request checks whether the credentials provided in the HTTP Basic header are valid. Returns the user ID and a bearer token for authorization in subsequent API calls. It is RECOMMENDED to implement this authentication method for non-public services only.",
+        "description": "This request checks whether the credentials provided in the HTTP Basic header are valid. Returns an access token to be used as Bearer token for authorization in subsequent API calls.\n\nIt is RECOMMENDED to implement this authentication method for non-public services only.\n\nAfter authentication, Clients SHOULD request further details about the user (e.g. the user id) from `GET /me`.",
```

Comment:
> After authentication, Clients SHOULD request further details

Why *should* clients do that? Currently only the `user_id` field of `/me` endpoint is required, which is typically not that useful for the end user.
I would just write "client can request further ..."

Reply:
This is related to #218. The user_id is required for the /files/... endpoints. If we remove the user_id from the /files/... endpoints, this can surely be a MAY or so. Otherwise I should explain why it *should* be requested.

Reply:
I added an explanation. If we agree on #218, I'll remove the corresponding parts from the API.